docs/api/communication-profiles/generic-mqtt-template.md

```diff
@@ -1,3 +1,175 @@
+
+---
+
+### Message Templates
+
+#### Request Message Template
+
+Messagees with request intent always have to follow the [MQTTRequestTemplate](../data-models/mqtt-request-template.md) data model in [JSON](https://datatracker.ietf.org/doc/html/rfc8259) format.
+
+#### Reponse Message Template
+
+Messages with response intent always have to follow the [MQTTResponseTemplate](../data-models/mqtt-response-template.md) data model in [JSON](https://datatracker.ietf.org/doc/html/rfc8259) format.
+
+#### Trace ID
+
+A string reference choosen by the requester party to identifiy the request message (not manadory). Service porviders are obliged to present the received trace id in the response messages.
+
+#### Authentication
+
+Authentication property must be used for providing identity or access related data if required in case of a given [Service](../../help/definitions.md#microservice-or-service). It is mandatory to specify it in IDDs if authentication or access validation is required by a given service instance.
+
+##### Identity
+
+If requester identity is required to be proven, then an [Identity Info](../../help/definitions.md#identity-info) or [Identiy Token](../../help/definitions.md#identity-token) should be provided in this property. The exact Identity data to be provided is depending on the applied [Authentication Policy](../authentication_policy.md).
+
+##### Access
+
+If the access right is required to be proven, then an [Access Token](../../help/definitions.md#access-token) should be provided in this property.
+
+#### Response Topic
+
+The topic on which the response is expected.
+
+#### QoS Requirement
+
+Required response [MQTT Quality of Service](https://www.hivemq.com/blog/mqtt-essentials-part-6-mqtt-quality-of-service-levels/).
+
+#### Payload
+
+The data to be transmitted to or from a given [service-operation](../../help/definitions.md#service-operation).
+
+#### Request Parameters
+
+The request parameters (alias params) are string key-value pairs to be transmitted to a given [service-operation](../../help/definitions.md#service-operation).
+
+#### Response Status
+
+Numerical values that indicates the result of a [service-operation](../../help/definitions.md#service-operation). It is mandatory to specify it in IDDs.
+
+#### Receiver
+
+Requester's identifier in the response message.
+
+## Examples
+
+### Request
+
+```
+Topic: <base-topic><delimiter><operation-name>
+QoS: <0|1|2>
+
+{
+   "traceId": "<trace-id>",
+   "authentication": "<access-token>",
+   "responseTopic": "<response-topic>",
+   "qosRequirement": <0|1|2>,
+   "params": {
+        "<key>": "<value>"
+   },
+   "payload": {
+      <data>
+   }
+}
+```
+
+```
+Topic: kelvinInfo/query-temperature
```

Comment:
It is very strange that you use the kelvinInfo service but the scale is celsius.